#! /bin/sh /usr/share/dpatch/dpatch-run
## 01-mdadm.conf-location.dpatch by martin f. krafft <madduck@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Reverses conffile locations to favour /etc/mdadm/mdadm.conf

@DPATCH@
diff -urNad mdadm-2.5.1~/Makefile mdadm-2.5.1/Makefile
--- mdadm-2.5.1~/Makefile	2006-06-26 00:06:29.000000000 +0200
+++ mdadm-2.5.1/Makefile	2006-06-26 00:06:45.936514898 +0200
@@ -48,8 +48,8 @@
 endif
 
 SYSCONFDIR = /etc
-CONFFILE = $(SYSCONFDIR)/mdadm.conf
-CONFFILE2 = $(SYSCONFDIR)/mdadm/mdadm.conf
+CONFFILE = $(SYSCONFDIR)/mdadm/mdadm.conf
+CONFFILE2 = $(SYSCONFDIR)/mdadm.conf
 MAILCMD =/usr/sbin/sendmail -t
 CONFFILEFLAGS = -DCONFFILE=\"$(CONFFILE)\" -DCONFFILE2=\"$(CONFFILE2)\"
 CFLAGS = $(CWFLAGS) $(CXFLAGS) -DSendmail=\""$(MAILCMD)"\" $(CONFFILEFLAGS)
diff -urNad mdadm-2.5.1~/ReadMe.c mdadm-2.5.1/ReadMe.c
--- mdadm-2.5.1~/ReadMe.c	2006-06-25 23:52:13.000000000 +0200
+++ mdadm-2.5.1/ReadMe.c	2006-06-26 00:06:34.108601730 +0200
@@ -504,7 +504,7 @@
 
 
 char Help_config[] =
-"The /etc/mdadm.conf config file:\n\n"
+"The /etc/mdadm/mdadm.conf config file:\n\n"
 " The config file contains, apart from blank lines and comment lines that\n"
 " start with a hash(#), four sorts of configuration lines: array lines, \n"
 " device lines, mailaddr lines and program lines.\n"
diff -urNad mdadm-2.5.1~/mdadm.8 mdadm-2.5.1/mdadm.8
--- mdadm-2.5.1~/mdadm.8	2006-06-25 23:52:13.000000000 +0200
+++ mdadm-2.5.1/mdadm.8	2006-06-26 00:06:34.116601671 +0200
@@ -225,9 +225,9 @@
 .TP
 .BR -c ", " --config=
 Specify the config file.  Default is to use
-.BR /etc/mdadm.conf ,
+.BR /etc/mdadm/mdadm.conf ,
 or if that is missing, then
-.BR /etc/mdadm/mdadm.conf .
+.BR /etc/mdadm.conf .
 If the config file given is
 .B partitions
 then nothing will be read, but
@@ -253,7 +253,7 @@
 permission to get any missing information, like component devices,
 array devices, array identities, and alert destination from the
 configuration file:
-.BR /etc/mdadm.conf .
+.BR /etc/mdadm/mdadm.conf .
 One exception is MISC mode when using
 .B --detail
 or
@@ -985,7 +985,7 @@
 or requested with (a possibly implicit)
 .B --scan.
 In the later case,
-.B /etc/mdadm.conf
+.B /etc/mdadm/mdadm.conf
 is used.
 
 If
@@ -1260,7 +1260,7 @@
 .B --scan
 will cause the output to be less detailed and the format to be
 suitable for inclusion in
-.BR /etc/mdadm.conf .
+.BR /etc/mdadm/mdadm.conf .
 The exit status of
 .I mdadm
 will normally be 0 unless
@@ -1296,7 +1296,7 @@
 then multiple devices that are components of the one array
 are grouped together and reported in a single entry suitable
 for inclusion in
-.BR /etc/mdadm.conf .
+.BR /etc/mdadm/mdadm.conf .
 
 Having
 .B --scan
@@ -1706,7 +1706,7 @@
 on Monitor mode.
 
 
-.SS /etc/mdadm.conf
+.SS /etc/mdadm/mdadm.conf
 
 The config file lists which devices may be scanned to see if
 they contain MD super block, and gives identifying information
diff -urNad mdadm-2.5.1~/mdadm.conf.5 mdadm-2.5.1/mdadm.conf.5
--- mdadm-2.5.1~/mdadm.conf.5	2006-06-25 23:52:13.000000000 +0200
+++ mdadm-2.5.1/mdadm.conf.5	2006-06-26 00:06:34.116601671 +0200
@@ -2,7 +2,7 @@
 .SH NAME
 mdadm.conf \- configuration for management of Software Raid with mdadm
 .SH SYNOPSIS
-/etc/mdadm.conf
+/etc/mdadm/mdadm.conf
 .SH DESCRIPTION
 .PP
 .B mdadm
diff -urNad mdadm-2.5.1~/mdassemble.8 mdadm-2.5.1/mdassemble.8
--- mdadm-2.5.1~/mdassemble.8	2006-06-25 23:52:13.000000000 +0200
+++ mdadm-2.5.1/mdassemble.8	2006-06-26 00:06:34.116601671 +0200
@@ -33,7 +33,7 @@
 
 .SH FILES
 
-.SS /etc/mdadm.conf
+.SS /etc/mdadm/mdadm.conf
 
 The config file lists which devices may be scanned to see if
 they contain MD super block, and gives identifying information
